docs/Writing-a-module.md

```diff
@@ -35,14 +34,13 @@ Every module surfaces on the admin settings page of the performance plugin, wher
 
 ## Example
 
-The following is a minimum module entry point file `/modules/my-module/load.php` (i.e. the module slug is "my-module"):
+The following is a minimum module entry point file `/modules/focus/my-module/load.php` (i.e. the module slug is "my-module"):
```

Comment:
Nit-pick, but maybe put an actual focus here such as `images`, to clarify we shouldn't just arbitrarily define focuses this way. This is different for the module slugs which can be any name.
```suggestion
The following is a minimum module entry point file `/modules/images/my-module/load.php` (i.e. the module slug is "my-module", and the focus is "images"):
```